lib/models/repository-states/present.js

```diff
@@ -233,17 +234,28 @@ export default class Present extends State {
         Keys.branches,
       ],
       // eslint-disable-next-line no-shadow
-      () => this.executePipelineAction('COMMIT', (message, options) => {
-        const opts = (!options || !options.coAuthors)
-          ? options
-          : {
-            ...options,
-            coAuthors: options.coAuthors.map(author => {
-              return {email: author.getEmail(), name: author.getFullName()};
-            }),
-          };
+      () => this.executePipelineAction('COMMIT', async (message, options) => {
+        const coAuthors = options && options.coAuthors;
+        const opts = !coAuthors ? options : {
+          ...options,
+          coAuthors: coAuthors.map(author => {
+            return {email: author.getEmail(), name: author.getFullName()};
+          }),
+        };
```

Comment:
@smashwilson can you double check to see if I retained the logic you intended with this change?